docs/apis/storm_compatibility.md

```diff
@@ -224,4 +224,12 @@ DataStream<String> rawInput = env.addSource(
 
 You can find more examples in Maven module `flink-storm-compatibilty-examples`.
 For the different versions of WordCount, see [README.md](https://github.com/apache/flink/tree/master/flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/README.md).
+To run the examples, you need to assemble a correct jar file.
+`flink-storm-compatibility-examples-0.10-SNAPSHOT.jar` is **no** valid jar file for job execution (it is only a standard maven artifact).
 
+There are example jars for embedded Spout and Bolt, namely `flink-storm-compatibility-examples-0.10-SNAPSHOT-WordCountSpoutSource.jar` and `flink-storm-compatibility-examples-0.10-SNAPSHOT-WordCountBoltTokenizer.jar`, respectively.
+Compare `pom.xml` to see how both jars are built.
+Furthermore, there is one example for whole Storm topologies (`flink-storm-compatibility-examples-0.10-SNAPSHOT-WordCountStorm.jar`).
+This jar is build using `maven-assembly-plugin` because it must contain `defaults.yaml` which cannot be included using `maven-jar-plugin`.
```

Comment:
This should also be possible with the maven-shade-plugin, for the sake of completeness.